src/display/api.js

```diff
@@ -513,6 +518,9 @@ async function _fetchDocument(worker, source, pdfDataRangeTransport, docId) {
     source.contentDispositionFilename =
       pdfDataRangeTransport.contentDispositionFilename;
   }
+  const transfers =
+    source.transferBinaryData && source.data ? [source.data.buffer] : null;
```

Comment:
Is there any reason to not transfer `initialData` ?

Reply:
It's already being transferred via the existing `PDFDataTransportStream` code, since that's really the only place where that option matters; see also the changes further down in this patch.

Reply:
Oh yes I missed that it wasn't used when calling `sendWithPromise`.